packages/sdk-node/tsup.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from 'tsup';
+
+import { name, version } from './package.json';
+
+console.log({ name, version });
+export default defineConfig(overrideOptions => {
+  const isProd = overrideOptions.env?.NODE_ENV === 'production';
+
+  return {
+    entry: ['src/index.ts', 'src/instance.ts'],
+    onSuccess: 'tsc',
+    clean: true,
+    // minify: isProd,
```

Comment:
🙃 Maybe remove this commented out line?

Reply:
Thanks Stef! Leaving as is for now, since we're going to align tsup config across all the packages soon.